converters/fromZigbee.js

```diff
@@ -1485,6 +1485,29 @@ const converters = {
             };
         },
     },
+    ias_enroll: {
+        cluster: 'ssIasZone',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const zoneState = msg.data.zoneState;
+            const iasCieAddr = msg.data.iasCieAddr;
+            const zoneId = msg.data.zoneId;
+            return {
+                enrolled: zoneState,
```

Comment:
This will be a number, but it makes more sense to expose a boolean instead (I think `enrolled: zoneState !== 0,`)

Reply:
Tested, Done.